packages/frontend/src/common/components/Header.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import Link from 'next/link'
+import NavigationItem from '@/common/components/NavigationItem'
+import Image from 'next/image'
+
+function Header() {
+  return (
+    <div className="sticky top-0 z-10 block bg-white h-[88px] shadow-lg flex justify-between px-6 items-center">
```

Comment:
If this is supposed to be 88px in height, we should ask Marek to change it maybe to either 80px or 96px (as is tailwind)

Reply:
Marek said 80px also looks good so I changed it to `h-20`